pom.xml

```diff
@@ -3,7 +3,7 @@
 
     <groupId>org.exist-db</groupId>
     <artifactId>exist-saxon-regex</artifactId>
-    <version>9.4.0-9.e2-SNAPSHOT</version>
+    <version>10.6-SNAPSHOT</version>
```

Comment:
Should probably be `10.6.e1`

The `eN` postfix is there to indicate the adjustments needed for eXist-db within the Saxon version.